### PR TITLE
Add notifications when batching complete.

### PR DESCRIPTION
**Pull request by kkamalov**

### CODE REVIEW

File: orchestra/communication/staffing.py [hunk 1]
```diff
@@ -109,5 +110,8 @@ def send_request_inquiries(staffbot, request, worker_batch_size):
 
     # check whether all inquiries have been sent out.
     if inquiries_sent < worker_batch_size:
+        notify_channel_slack(request.task.project.slack_group_id,
+                             ('{} staffing request inquiries sent'
```
Comment (joshblum) [inline orchestra/communication/staffing.py]: `'All staffing requests for task {} have been sent!'.format(request.task.id)`

**Comment by joshblum** [comment]
one comment, then lgtm
TODO: notify when accepted and when all rejected

File: orchestra/utils/notifications.py [hunk 2]
```diff
@@ -160,3 +161,10 @@ def _notify_experts_slack_status_change(task, current_worker):
                                 task.project.slack_group_id,
                                 with_slack_link=False,
                                 with_user_mention=True)
+
+@run_if('ORCHESTRA_SLACK_EXPERTS_ENABLED')
+def notify_channel_slack(slack_channel, text):
```
Comment (marcua) [inline orchestra/utils/notifications.py]: message_experts_slack_group

File: orchestra/utils/notifications.py [hunk 3]
```diff
@@ -160,3 +161,10 @@ def _notify_experts_slack_status_change(task, current_worker):
...
+    slack = OrchestraSlackService(settings.SLACK_EXPERTS_API_KEY)
+    slack_channel = task.project.slack_group_id,
```
Comment (marcua) [inline orchestra/utils/notifications.py]: task is not an arg

**Pull request merged**